Promact.Oauth.Server/src/Promact.Oauth.Server/Models/AppSettings.cs

```diff
@@ -10,12 +10,10 @@ public class AppSettings
 
         public string Host { get; set; }
 
-        public int Port { get; set; }
+        public string Port { get; set; }
```

Comment:
@chintans When i set to int, it throws error **Additional information: Failed to convert '587;' to type 'System.Int32'.** when constructor is loaded and fetch appseetings value beacause I think appseeting return the string values.